src/components/HOC/withScore.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+const withScore = (WrappedComponent, charactersQuantity) => {
+    class WithScore extends React.Component {
+        constructor(props) {
+            super(props)
+            this.state = {
+                score: 0,
+                currentTry: 1,
+                scoreDisplay: false,
+                scoreMessage: '',
+                scoreStyle: 'font--small--red',
+            }
+        }
+
+        toggleScoreDisplay = () => {
+            this.setState({
+                scoreDisplay: !this.state.scoreDisplay
+            });
+        }
+
+        increment = (stateValue) => {
```

Comment:
This function is very dangerous.
What if I don;t know the implementation of it and I will decide to use it like that:
`increment('scoreDisplay')`

Reply:
I've used it only in HOC to generate two other methods which I pass down to components, hope now it is fine...